api/types/session_tracker.go

```diff
@@ -99,6 +99,15 @@ type SessionTracker interface {
 	// GetKubeCluster returns the name of the kubernetes cluster the session is running in.
 	GetKubeCluster() string
 
+	// GetDesktopName returns the name of the Windows desktop the session is running in.
+	GetDesktopName() string
```

Comment:
weird, i don't see this being a duplicate as `github actions/analyze` says it is...